storage/migrations/sqlite3/1_init.sql

```diff
@@ -174,8 +174,7 @@ CREATE TABLE IF NOT EXISTS "deliveries" (
 );
 
 CREATE TABLE IF NOT EXISTS "send_logs" (
-  "id"            integer primary key autoincrement,
-  "uuid"          varchar(36) unique NOT NULL,
+  "uid"           varchar(100) primary key,
```

Comment:
We should just name this `id`. Also, if we add it to be varchar(27) would it make problems? I want it to be consistent with the mysql table.

Reply:
And  i think we should remove the alter table since app is not on production, change it in the first sql file?
@FilipNikolovski @gudgl

Reply:
@Dzalevski  If i change init file i need to drop all my table.